core/services/ocr2/plugins/functions/config/config.go

```diff
@@ -17,6 +18,8 @@ type PluginConfig struct {
 	PruneBatchSize                  uint32                     `json:"pruneBatchSize"`
 	ListenerEventHandlerTimeoutSec  uint32                     `json:"listenerEventHandlerTimeoutSec"`
 	MaxRequestSizeBytes             uint32                     `json:"maxRequestSizeBytes"`
+	AllowlistBlockConfirmations     int64                      `json:"allowlistBlockConfirmations"`
```

Comment:
For the allowlist to work you need to run UpdatePeriodically() or UpdateFromContract() in a loop. Check out functions.handler.go.

Reply:
This is not good. İ would wish the service (Allowlist) would be self-contained. E.g. implementing start/stop and hide all this logic internally.

Reply:
Yeah, I should've done that. I will convert to Start()/Stop() and own goroutine but in the meantime you can just call UpdatePeriodically() in your handler's Start().

Reply:
I reworked the Allowlist.